package.json

```diff
@@ -77,6 +78,7 @@
     "karma-sinon": "^1.0.4",
     "load-grunt-config": "^0.19.2",
     "mocha": "^3.2.0",
+    "raf": "^3.4.0",
```

Comment:
why do we need this lib/polyfill?

Reply:
raf is a polyfill for `requestAnimationFrame` api. React 16 uses requestAnimationFrame now, because in the future they will enable async rendering.

Reply:
In case of testing, if you downloaded the library code and install all the dependencies, you probably will have react v16 instead of react v15. When running test with grunt, you'll find an error related to requestAnimationFrame. That's why I have added the polyfill!